src/styles/utilits/mixin.scss

```diff
@@ -0,0 +1,13 @@
+@mixin spotSize ($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin PositionArrov ($hand-height) {
+  top: calc(50% - #{$hand-height});
+  left: calc(50% - (#{$hand-width} / 2));
+}
+
+
+
```

Comment:
we have to place in the center of screen.

Reply:
done )))